src/main/java/com/liveramp/daemon_lib/builders/BaseForkingDaemonBuilder.java

```diff
@@ -67,6 +67,7 @@ public E addToConfigStorageDeserializers(Function<byte[], ? extends T> deseriali
   @NotNull
   @Override
   protected JobletExecutor<T> getExecutor() throws IllegalAccessException, IOException, InstantiationException {
+    deserializers.add(JobletConfigStorage.getDefaultDeserializer());
```

Comment:
Do you think the `getExecutor` method should have side effects? Maybe we shouldn't add to the existing instance variable but rather instantiate a new list that concatenates the two. In addition lets rename this to `customDeserializers` if we go with this approach.